libse/SubtitleFormats/Son.cs

```diff
@@ -56,32 +51,24 @@ private static string EncodeTimeCode(TimeCode time)
         public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string fileName)
         {
             //0001  00:00:19:13 00:00:22:10 a_0001.tif
+            var regexTimeCodes = new Regex(@"^\d\d\d\d\t+(\d\d:\d\d:\d\d:\d\d)\t(\d\d:\d\d:\d\d:\d\d)\t.+\.(?:tif|tiff|png|bmp|TIF|TIFF|PNG|BMP)", RegexOptions.Compiled);
             Paragraph p = null;
             subtitle.Paragraphs.Clear();
             _errorCount = 0;
-            var regexTimeCodes = new Regex(@"^\d\d\d\d[\t]+\d\d:\d\d:\d\d:\d\d\t\d\d:\d\d:\d\d:\d\d\t.+\.(tif|tiff|png|bmp|TIF|TIFF|PNG|BMP)", RegexOptions.Compiled);
             int index = 0;
             foreach (string line in lines)
             {
-                if (regexTimeCodes.IsMatch(line))
+                var match = regexTimeCodes.Match(line);
+                if (match.Success)
                 {
-                    string temp = line.Substring(0, regexTimeCodes.Match(line).Length);
-                    string start = temp.Substring(5, 11);
-                    string end = temp.Substring(12 + 5, 11);

```

Comment:
Fixed start indices (`5` and `12+5`)  will be incorrect if index and start time-code are separated by more than one HT.